core/src/main/java/org/mule/runtime/core/internal/store/PartitionedPersistentObjectStore.java

```diff
@@ -50,7 +50,7 @@ public PartitionedPersistentObjectStore(MuleContext context) {
   }
 
   @Override
-  public synchronized void open() throws ObjectStoreException {
+  public void open() throws ObjectStoreException {
```

Comment:
are you sure of this?

Reply:
Not, my bad. That is not supposed to be changed.